### PR TITLE
Update Cucumber FAQ

### PR DESCRIPTION
**Pull request by conversayShawn**
Add some verbiage around cucumber

### CODE REVIEW

File: docs/faq/questions/using-cypress-faq.mdx [hunk 1]
```diff
@@ -1109,10 +1111,22 @@ JavaScript Cypress tests.
 - try using the
   [Cucumber preprocessor](https://github.com/badeball/cypress-cucumber-preprocessor)
   and search our [Plugins](/plugins) page for additional helper plugins
+
+## <Icon name="angle-right" /> Should I use Cucumber to write tests?
```
Comment (emilyrohrbough) [inline docs/faq/questions/using-cypress-faq.mdx:1115]: Adding a new section under `Can I use Cucumber to write tests?` feels weird. I'd just combine the answer with the above response:
```suggestion
```

File: docs/faq/questions/using-cypress-faq.mdx [hunk 2]
```diff
@@ -1109,10 +1111,22 @@ JavaScript Cypress tests.
...
+
+Cypress does not currently have an official stance on Cucumber as each use case
+is unique and based on the needs of the user.
```
Comment (emilyrohrbough) [inline docs/faq/questions/using-cypress-faq.mdx:1118]:
```suggestion
Deciding to use Cucumber is based on the needs of the users.
```

File: docs/faq/questions/using-cypress-faq.mdx [hunk 3]
```diff
@@ -479,6 +479,8 @@ arguments, JSON files and other means.
 Yes.
 [You can override this with `userAgent` in your Cypress configuration](/guides/references/configuration#Browser).
 
+_Note_
```
Comment (emilyrohrbough) [inline docs/faq/questions/using-cypress-faq.mdx:482]:
```suggestion
```

**Review by elylucas — CHANGES_REQUESTED** [review]
I agree with Emily's comment about not having a second question. Also was looking at the answer for the first question and I think we could summarize it all with something like:

While Cypress does not provide official support for Cucumber, you can use Cucumber through a [community plugin](link to the plugin). Using the plugin adds additional complexity to your testing workflow, so ensure your team understands the benefits and drawbacks before adopting Cucumber. 

If your team is looking to do behavior driven development, you might be interested in these articles that discuss using BDD's given/when/then syntax directly in Cypress instead of using Cucumber's scenarios:

(links you have to the blogs)

File: docs/faq/questions/using-cypress-faq.mdx (hunk 3 above)
Comment (conversayShawn) [reply docs/faq/questions/using-cypress-faq.mdx]: 👍🏽

**Review by emilyrohrbough — APPROVED** [review]

**Review by elylucas — APPROVED** [review]

**Pull request merged**